src/BiometryService/BiometryService.csproj

```diff
@@ -7,7 +7,7 @@
 		<Company>nventive</Company>
 		<AssemblyName>BiometryService</AssemblyName>
 		<PackageId>BiometryService</PackageId>
-		<Description>BiometryService</Description>
+		<Description>Enable biometry across Android, iOS and Windows (UWP and WinUI).</Description>
```

Comment:
You need to update the metadata on the other csproj too (the Abstractions project).